### DISCUSSION TITLE
SPI_RX_READY; SPI_TX_PENDING

### DISCUSSION THREAD

**Discussion by Bobbybigears** [Sevenstax/FreeV2G#121] [Q&A]
Hi,
With regard to using the SPI_RX_READY and SPI_TX_PENDNG pins flags with the Whitebeet SPI host protocol, what do they actually mean with regard to buffer space etc. For example does SPI_RX_READY going high mean there's 1-byte of free space on the RX QUEUE or 500 bytes? Similarily, at what level of available queue space does SPI_TX_PENDING go high?

Cheers Rob

**Comment by jpo-stx** [comment]
Hi @Bobbybigears ,
When the RX_READY line is high this means that up to 1500 bytes can be transmitted by the host.
TX_PENDING goes high when there is at least one complete framing message to be sent. 
I will add this information to the manual.
